pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -90,6 +90,9 @@ static int _format(vfs_mount_t *mountp)
 
     /* make sure the volume has been initialized */
     if (_init(mountp)) {
+        if (!IS_USED(CONFIG_FATFS_FORMAT_ALLOC_STATIC)) {  
+            free(work);  
+        } 
```

Comment:
The Ci still does not like the added whitespace at the end of lines

```suggestion
        if (!IS_USED(CONFIG_FATFS_FORMAT_ALLOC_STATIC)) {
            free(work);
        }
```